ruby/vampires.rb

```diff
@@ -0,0 +1,139 @@
+#Initialize required variables
+full_name = ""
+age = 0
+birth_year = 0
+correct_age = false
+allergy = ""
+serve_garlic = false
+health_insurance = false
+num_employees = 0
+
+#Compares age to birth year to see if age is accurate
+def compare_age(age, birth_year)
+  if (2017 - age) == birth_year
```

Comment:
When checking something like a year consider an approach that keeps us from needing to change our code every new year.

Take a look the Ruby `Time` class.  `Time.new.year`